WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationRepository.swift

```diff
@@ -37,10 +37,21 @@ public protocol ConversationRepositoryProtocol {
     /// - returns : The MLS group ID.
 
     func pullMLSOneToOneConversation(
-        userID: String,
+        for userID: String,
         domain: String
     ) async throws -> String
 
+    /// Fetches or creates a conversation locally.
+    /// - Parameters:
+    ///     - id: The conversation ID.
+    ///     - domain: The conversation domain if any.
+    /// - Returns: The `ZMConversation` found or created locally.
+
+    func fetchOrCreateConversation(
+        with id: UUID,
```

Comment:
I think `id` here is perhaps more descriptive from the outside. From https://www.swift.org/documentation/api-design-guidelines/#naming:

> Compensate for weak type information to clarify a parameter’s role. Especially when a parameter type is NSObject, Any, AnyObject, or a fundamental type such as Int or String, type information and context at the point of use may not fully convey intent. In this example, the declaration may be clear, but the use site is vague.

```swift
grid.add(self, for: graphics) // vague
grid.addObserver(self, forKeyPath: graphics) // clear
```

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2091/commits/cbebb8686ef60bc47af686531cbc8732218095fe